src/GraphODataTemplateWriter/CodeHelpers/Android/TypeHelperAndroid.cs

```diff
@@ -37,6 +37,13 @@ public static string GetTypeString(this OdcmType @type)
                     return "com.microsoft.graph.model.DateOnly";
                 case "Binary":
                     return "byte[]";
+                case "Stream":
```

Comment:
This is correct - this relevant exclusion can be found in [`BaseModel.template.tt#CreatePropertyDef`](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/9c434fc35e09fecc2355469d7fd38edd6aef69db/Templates/Android/BaseModel.template.tt#L637). The deferred implementation is [here](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/9c434fc35e09fecc2355469d7fd38edd6aef69db/Templates/Android/generated/BaseEntityRequestBuilder.java.tt#L96-L98), with its interface found [here](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/9c434fc35e09fecc2355469d7fd38edd6aef69db/Templates/Android/generated/IBaseEntityRequestBuilder.java.tt#L70).